src/main/java/se/liu/ida/hefquin/engine/wrappers/graphqlwrapper/impl/RdfViewConfigurationImpl.java

```diff
@@ -0,0 +1,139 @@
+package se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.impl;
+
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.RdfViewConfiguration;
+import se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.data.GraphQLEntrypoint;
+import se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.data.GraphQLViewProperty;
+import se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.data.impl.GraphQLEntrypointType;
+import se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.data.impl.GraphQLFieldType;
+
+public class RdfViewConfigurationImpl implements RdfViewConfiguration {
+
+    // Maps classnames to a map which in turn maps property names to a GraphQLViewProperty object
+    protected final Map<String, Map<String,GraphQLViewProperty>> classToProperty;
+
+    // Maps classnames to a map which in turn maps a GraphQLEntrypointType to a GraphQLEntrypoint
+    protected final Map<String, Map<GraphQLEntrypointType,GraphQLEntrypoint>> classToEntrypoint;
+
+    // The URI prefixes used by the virtual RDF view
+    protected final String classPrefix = "c:";
+    protected final String propertyPrefix = "p:";
```

Comment:
Maybe, for the sake of extensibility, these prefixes should be customizable, and have `c:` and `p:` as the default values. Please add constructor parameters for these.
Also, consider using full URI notation. This means something like `http://example.org/c/` instead of just `c:`.